File: Java/Practice/LeetCode/Medium/Partitioning Into Minimum Number Of Deci-Binary Numbers.java
// https://leetcode.com/problems/partitioning-into-minimum-number-of-deci-binary-numbers/

class Solution {
    public int minPartitions(String n) {
        int max = 0;

        for (int i = 0; i < n.length(); i++) {
            int digitValue = Character.getNumericValue(n.charAt(i));

            if (max < digitValue) {
                max = digitValue;
            }
        }

        return max;  
    }
}

File: TypeScript/Practice/LeetCode/Medium/Partitioning Into Minimum Number Of Deci-Binary Numbers.ts
// https://leetcode.com/problems/partitioning-into-minimum-number-of-deci-binary-numbers/

function minPartitions(n: string): number {
    let max: number = 0;

    for (let i = 0; i < n.length; i++) {
        if (max < Number(n[i])) {
            max = Number(n[i]);
        }
    }

    return max; 
};